src/plugins/intel_gna/tests/functional/pass_tests/broadcast_tile_issue.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ngraph/opsets/opset9.hpp>
+
+#include "shared_test_classes/base/layer_test_utils.hpp"
+
+class BroadcastToTileIssueAfterAddActivation : public LayerTestsUtils::LayerTestsCommon {
+public:
+    void SetUp() override;
+};
+
+void BroadcastToTileIssueAfterAddActivation::SetUp() {
+    std::vector<size_t> input_shape = {1, 590};
+    auto net_precision = InferenceEngine::Precision::FP32;
+    targetDevice = CommonTestUtils::DEVICE_GNA;
+
+    auto ng_prc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(net_precision);
+    auto input_param = std::make_shared<ngraph::opset9::Parameter>(ng_prc, ngraph::Shape{input_shape});
```

Comment:
Can we also set some friendly name for Parameter and Result and checks that they are still present in topology after the transformation?

Reply:
done